### PR TITLE
base: u-boot-ostree-scr-fit: move boot-common-imx to bsp

### PR DESCRIPTION
**Pull request by igoropaniuk**
Move i.MX-specific boot-common-imx.cmd.in to bsp sublayer.

Signed-off-by: Igor Opaniuk <igor.opaniuk@foundries.io>

### CODE REVIEW

**Review by quaresmajose — APPROVED** [review]
LGTM

File: meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit.bb [hunk 1]
```diff
@@ -27,8 +26,11 @@ inherit deploy
 do_configure[noexec] = "1"
 
 do_compile() {
-	sed -e '/@@INCLUDE_COMMON_IMX@@/ {' -e 'r ${S}/boot-common-imx.cmd.in' -e 'd' -e '}' \
-			"${S}/boot.cmd" > boot.cmd.in
+	# Check if any modifications were done by bsp specific
+	# do_compile() prepends from meta-lmp-bsp
+	if [ ! -f boot.cmd.in ]; then
+		cp "${S}/boot.cmd" boot.cmd.in
```
Comment (mike-scott) [inline meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit.bb:32]: The comment is sort of backwards.  This handles a case where there is no BSP modification and we're using the original `boot.cmd`

Comment (igoropaniuk) [reply meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit.bb:32]: fixed

Comment (mike-scott) [reply meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit.bb:32]: perfect thanks!

**Review by mike-scott — APPROVED** [review]

**Pull request merged**